config/routes.rb

```diff
@@ -2,9 +2,11 @@
 
   get "/" => "home#index"
 
+  get "/users/search" => "users#search"
```

Comment:
you can include this route like this http://guides.rubyonrails.org/routing.html#adding-collection-routes